src/background.js

```diff
@@ -260,6 +262,11 @@ function updateWakeAndBookmarks() {
     });
 }
 
+function startPeriodicAlarm() {
+  log('starting periodic alarm');
+  return browser.alarms.create(PERIODIC_ALARM_NAME, { periodInMinutes: 1 });
```

Comment:
Will the `return browser.alarms.clearAll()` at line 241/243 clear this alarm, too? And if so, do we need to re-create it in that function?

Reply:
Oof yeah, I think that's right. Will look at that

Reply:
Pushed a tweak to just specifically clear WAKE_ALARM_NAME and not the periodic alarm.